draft-rha-jose-hpke-encrypt.md

```diff
@@ -97,27 +102,16 @@ JOSE-native security mechanisms or by one of the authenticated variants of HPKE.
 
 This document defines the use of the HPKE with JOSE.
 
-
 --- middle
 
 # Introduction
 
 Hybrid public-key encryption (HPKE) {{RFC9180}} is a scheme that 
 provides public key encryption of arbitrary-sized plaintexts given a 
-recipient's public key. HPKE utilizes a non-interactive ephemeral-static 
-Diffie-Hellman exchange to establish a shared secret. The motivation for
```

Comment:
specifically we do not want to mention DH here, not all KEMs are DH Kems

Reply:
Agree. We should be forward-looking.

Reply:
Agreed.